src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -691,6 +691,71 @@ public class C
             }
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        [WorkItem(52534, "https://github.com/dotnet/roslyn/issues/52534")]
+        public async Task SuggestParameterNamesFromExistingOverloads()
+        {
+            var markup = @"
+using System.Threading;
+public class C
+{
+    void M(CancellationToken myTok) { }
+
+    void M(CancellationToken $$
+}
+";
+            await VerifyItemExistsAsync(markup, "myTok", glyph: (int)Glyph.Parameter);
+            await VerifyItemExistsAsync(markup, "cancellationToken", glyph: (int)Glyph.Parameter);
```

Comment:
can you add an assert about what is selected?

Reply:
@CyrusNajmabadi What's the API for doing so?

Reply:
i think you'll need to add a test to CompletionCommandHandlerTests

Reply:
Added the test.